package.json

```
@@ -5,11 +5,12 @@
   "scripts": {
     "predev": "check-node-version --package",
     "dev": "next dev",
-    "build": "next build",
+    "build": "next build && next export",
```

Can we create a separate "export" script that does this? We can leave the "build" script as it was (it'll play nice with Netlify that way).

```suggestion
    "build": "next build",
    "export": "next build && next export",
```